config-default.yml

```diff
@@ -196,10 +196,17 @@ guild:
         mod_announcements:      &MOD_ANNOUNCEMENTS      372115205867700225
         admin_announcements:    &ADMIN_ANNOUNCEMENTS    749736155569848370
 
-        # Voice
-        code_help_voice:                    755154969761677312
-        code_help_voice_2:                  766330079135268884
+        # Voice Chat
+        code_help_chat:                     755154969761677312
+        code_help_chat_2:                   766330079135268884
         voice_chat:                         412357430186344448
+        admins_voice_chat:                  000000000000000000 # FIXME
+        staff_voice_chat:                   541638762007101470
+
+        # Voice Channels
+        code_help_voice:                    751592231726481530
```

Comment:
I think `code_help_voice_1` may be a better name for this, following suite with the other code help voice channel.

Reply:
Agreed, I just left it in the original format to minimize changes, but I've updated it.